amber-slsa-buildtype.json

```diff
@@ -0,0 +1,137 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Amber SLSA buildType",
+  "description": "Describes a SLSA buildType, as defined in https://slsa.dev/provenance/v0.2.",
+  "type": "object",
+  "required": [
+    "buildType",
+    "buildConfig",
+    "materials"
+  ],
+  "not": {
+    "required": [
+      "invocation"
+    ]
+  },
+  "properties": {
+    "buildType": {
+      "const": "https://github.com/project-oak/transparent-release/amber-slsa-buildtype.json"
```

Comment:
If we want to hash the buildType (see https://github.com/in-toto/attestation/issues/81 for context), we could ask clients to specify this as a [ni URL](https://www.rfc-editor.org/rfc/rfc6920.html). Using a format that is also a valid HTTP URL would be nice for human readability eg `ni://example.com/sha-256;f4OxZX_x_FO5LcGBSKHWXfwtSx-j1ncoSt3SABJtkGk`. But it seems difficult to create a conformant HTTP link for inside a github repo.

Reply:
Also worth noting that with hashes there's a bit of a general inconvenience that this schema file cannot include its own hash.

Reply:
Actually, I don't see what is the benefit of a digest when the buildType is a schema like this. It is very easy to validate a json file against the schema. If the provider silently modifies the schema, it will easily be detected as the validation will fail. So, in case of a URI pointing to a json schema as the buildType, the consumer of the attestation does not have to trust the author of the schema. 

Let's not worry about the digests, at least for this version, and just use a URI.

Reply:
Happy to skip validation for this version. :) I do think it still matters for JSON schema's. While the provider cannot change the structure, they can change what each key means. :)